src/visitors/visitor_utils.cpp

```diff
@@ -260,4 +260,19 @@ std::pair<std::string, std::unordered_set<std::string>> statement_dependencies(
     return {key, out};
 }
 
+std::string get_indexed_name(const ast::IndexedName& node) {
```

Comment:
shouldn't this be a method of the class `IndexedName`?

Reply:
I think that this is just a pain to do because of the need of `to_nmodl` function. `get_length` returns an `Expression` that cannot be easily converted to `string` since it might have various different forms. I'm not sure if it's worth it to complicate the current code that much